perception/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -82,8 +83,15 @@ TrackerDebugger::TrackerDebugger(rclcpp::Node & node) : node_(node), last_input_
         "debug/tentative_objects", rclcpp::QoS{1});
   }
 
-  // initialize stop watch
+  // initialize stop watch and diagnostics
   startStopWatch();
+  setupDiagnostics();
+}
+
+void TrackerDebugger::setupDiagnostics()
+{
+  diagnostic_updater_.setHardwareID(node_.get_name());
+  diagnostic_updater_.add("Delay Check", this, &TrackerDebugger::checkDelay);
```

Comment:
[imo]
should set update duration here. see 
https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/pointcloud_preprocessor/src/outlier_filter/dual_return_outlier_filter_nodelet.cpp#L65C1-L69